test/conformance/README.md

```diff
@@ -0,0 +1,16 @@
+# Conformance tests
+
+Conformance tests verifies kantive eventing implementation for expected behavior described in
+[specification](https://github.com/knative/eventing/tree/master/docs/spec).
+
+## Running performance tests
+
+Run test with e2e tag and optionally select conformance test
```

Comment:
@Fredy-Z should we use the `e2e` tag? Or something else, like `conformance`?

Reply:
Serving is also using `e2e` tag for their conformance tests, I think we can follow the same pattern.

Reply:
I followed that pattern